elastic/elasticsearch
# fix monitoring-beats-mb template

### Summary
Context https://github.com/elastic/apm-server/issues/7139

Moves the `beats_stats.apm-server` mapping section under the `beats_stats.metrics` property as expected by Stack Monitoring.

### Testing
- start local elasticsearch with updated mappings
- install apm integration assets
- start apm-server
- ingest some data to the APM Server, run this Go program[1] a few times (Export the ELASTIC_APM_SERVER_URL environment variable)
- verify Stack monitoring's Apm server graphs are populated[2]

[1]
```
package main

import (
	"context"
	"fmt"
	"os"
	"time"

	"go.elastic.co/apm"
)

func main() {
	version := "undefined"
	if len(os.Args) > 1 {
		version = os.Args[1]
	}
	name := fmt.Sprintf("apm-server-%s", version)
	// apm.DefaultTracer.SetSampler(apm.NewRatioSampler(0.0))

	for i := 0; i < 1000; i++ {
		tx := apm.DefaultTracer.StartTransaction(name, "type")
		ctx := apm.ContextWithTransaction(context.Background(), tx)
		span, ctx := apm.StartSpan(ctx, name, "type")
		span.Duration = time.Second
		span.End()
		tx.Duration = 2 * time.Second
		tx.End()
		<-time.After(time.Millisecond)
	}
	<-time.After(time.Second)
	apm.DefaultTracer.Flush(nil)
	fmt.Printf("%s: %+v\n", name, apm.DefaultTracer.Stats())
}
```

[2]
<img width="1743" alt="Screenshot 2022-01-26 at 17 04 32" src="https://user-images.githubusercontent.com/5239883/151202793-7c9d34f8-4ec8-4475-a712-817703196a70.png">
